### ISSUE TITLE
How to save validation-loss in tensorboard using scepter framework? 😭

### ISSUE THREAD

**Issue by chenyg59** [ali-vilab/ACE_plus#29]
I want to save the validation-loss during training, but the code now only save the training-loss? I am not familiar with the scepter framework and have been trying for a long time without success? Can someone kindly guide me?😭

**Comment by maochaojie** [comment]
We provide a way to record validation loss in the FFT parameter configuration, using ValLossHook to compute and record the validation loss. You can refer to the following config "https://github.com/ali-vilab/ACE_plus/blob/main/train_config/ace_plus_fft.yaml"  to understand the details.

![Image](https://github.com/user-attachments/assets/1d23e151-8083-4887-87b7-7a69cae78866)
